### DISCUSSION TITLE
How can I create a new tab with specific command to be ran?

### DISCUSSION THREAD

**Discussion by Rid1FZ** [kovidgoyal/kitty#6578] [Q&A]
I use Distrobox which creates a new desktop entry every time I create a new container. It uses my default terminal to start my container. But the problem is, If I create a new tab, instead of using the previous command which was used to launch kitty, it opens my host's shell. I want it to use the command I used very first to run a new kitty session. How can I achieve this?

**Comment by kovidgoyal** [comment]
I dont know what youa re asking exactly, but to create new tabs in existing kitty instances use either remote control or kitty --single-instance. Search for both in the kitty docs to find more details.
